### PR TITLE
bench-tps: Increase default lamports-per-account to 10 SOL

### LINKED ISSUE #22559: Set minimum lamports requirement for stake accounts
## Problem

Stake accounts do not have a minimum lamports requirement. This may cause the number of stake accounts to be higher than necessary, which affects both #20384 and #21694.

## Proposed Solution

Require a minimum balance for stake accounts. With #22292, this will raise the minimum balance to be rent-exempt. I feel the minimum balance for stake accounts should be much higher (i.e. >= 1 sol).

#### Creating accounts

This is straight forward; in order to create a new stake account, the new balance must be greater-than-or-equal-to the minimum balance.

#### Transfers

TODO: How should/can transfers be handled? Should the minimum balance still be enforced here? How does this interact with staked vs non-staked balances?

### Ethereum

Eth 2 requires 32 ETH to stake to be a validator. Smaller amounts of eth can be staked in stake pools. [link](https://ethereum.org/en/eth2/staking/)

### Avalanche

The minimum amount required to stake to a delegator is 25 AVAX (not counting pools), and 2000 AVAX to be a validator. [link](https://docs.avax.network/learn/platform-overview/staking/)

### Polkadot

From what I can tell, _nominating_ has a minimum of 80 DOT ([about to be 120 DOT](https://polkadot.polkassembly.io/referendum/34)), which I believe is like staking to a delegator. [link](https://wiki.polkadot.network/docs/learn-staking)

### Fantom

Staking on Fantom has a 1 FTM minimum. [link](https://docs.fantom.foundation/staking/how-to-stake-on-fantom)

### PR DESCRIPTION
**Pull request by brooksprumo**
#### Problem

Bench-tps fails with default stake amount because it is too low. See linked issue for more context.

#### Summary of Changes

Increase default stake amount.

Fixes #22559

### CODE REVIEW

**Review by CriesofCarrots — COMMENTED** [review]
The test changes look fine.

**Comment by CriesofCarrots** [comment]
Also, do you mean for this to close #22559 ?

File: bench-tps/src/cli.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ use {
     std::{net::SocketAddr, process::exit, time::Duration},
 };
 
-const NUM_LAMPORTS_PER_ACCOUNT_DEFAULT: u64 = solana_sdk::native_token::LAMPORTS_PER_SOL;
+const NUM_LAMPORTS_PER_ACCOUNT_DEFAULT: u64 = 10 * solana_sdk::native_token::LAMPORTS_PER_SOL;
```
[the remark this answers is not in the document]
Comment (brooksprumo) [reply bench-tps/src/cli.rs:13]: Ah gotcha. I thought I may've traced this back to eventually creating a new stake account that was then delegating, but I probably mixed myself up. I'm going to close this PR for now then; if I see bench-tps issues after #25365 goes in, I can reopen this one.

**Comment by CriesofCarrots** [comment]
> > Also, do you mean for this to close #22559 ?
> 
> Originally yes, since that issue was for bench-tps and the net scripts, but I have changed the issue.

#22559  is your big raise-delegations issue. Just didn't want you to close it inadvertently!

Screen Shot 2022-05-19 at 12 03 56 PM

**Comment by brooksprumo** [comment]
> > > Also, do you mean for this to close #22559 ?
> > 
> > Originally yes, since that issue was for bench-tps and the net scripts, but I have changed the issue.
> 
> #22559 is your big raise-delegations issue. Just didn't want you to close it inadvertently!

Ah, thanks for catching!

**Pull request closed without merging**